### PR TITLE
Issue 30 - Little fix for IE9

### PR DESCRIPTION
**Pull request by rodfersou**

### CODE REVIEW

File: dist/getUserMedia.js [hunk 1]
```diff
@@ -81,7 +81,7 @@
                     // Fallback to flash
                     var source, el, cam;
 
-                    source = '<object id="XwebcamXobjectX" type="application/x-shockwave-flash" data="' + options.swffile + '" width="' + options.width + '" height="' + options.height + '"><param name="movie" value="' + options.swffile + '" /><param name="FlashVars" value="mode=' + options.mode + '&amp;quality=' + options.quality + '" /><param name="allowScriptAccess" value="always" /></object>';
+                    source = '<object id="XwebcamXobjectX" classid="clsid:d27cdb6e-ae6d-11cf-96b8-444553540000" type="application/x-shockwave-flash" data="' + options.swffile + '" width="' + options.width + '" height="' + options.height + '"><param name="movie" value="' + options.swffile + '" /><param name="FlashVars" value="mode=' + options.mode + '&amp;quality=' + options.quality + '" /><param name="allowScriptAccess" value="always" /></object>';
```
Comment (rodfersou) [inline dist/getUserMedia.js:84]: The fix is about to add the classid attribute (the ID is the same everywhere), and this makes IE9 load faster the flash object.

File: lib/getUserMedia.js [hunk 2]
```diff
@@ -78,7 +78,7 @@
                     // Fallback to flash
                     var source, el, cam;
 
-                    source = '<object id="XwebcamXobjectX" type="application/x-shockwave-flash" data="' + options.swffile + '" width="' + options.width + '" height="' + options.height + '"><param name="movie" value="' + options.swffile + '" /><param name="FlashVars" value="mode=' + options.mode + '&amp;quality=' + options.quality + '" /><param name="allowScriptAccess" value="always" /></object>';
+                    source = '<object id="XwebcamXobjectX" classid="clsid:d27cdb6e-ae6d-11cf-96b8-444553540000" type="application/x-shockwave-flash" data="' + options.swffile + '" width="' + options.width + '" height="' + options.height + '"><param name="movie" value="' + options.swffile + '" /><param name="FlashVars" value="mode=' + options.mode + '&amp;quality=' + options.quality + '" /><param name="allowScriptAccess" value="always" /></object>';
```
Comment (rodfersou) [inline lib/getUserMedia.js:81]: The fix is about to add the classid attribute (the ID is the same everywhere), and this makes IE9 load faster the flash object.

**Comment by rodfersou** [comment]
It looks like travis is not working properly...

**Comment by addyosmani** [comment]
Thanks for the fixes. These LGTM. Travis is complaining due to some configuration settings I need to refresh in the backend. Will take care of that. Cheers!

**Comment by rodfersou** [comment]
@addyosmani you are welcome! :)

**Pull request merged**